spec/system/nsm/hearing_details/suggestion_autocomplete_spec.rb

```diff
@@ -14,12 +14,12 @@
     click_on 'Save and continue'
 
     fill_in 'Which court was the last case hearing heard at?', with: 'Aldershot'
-    expect(page).to have_field('Which court was the last case hearing heard at?', with: "Aldershot Magistrates' Court")
+    expect(page).to have_field('Which court was the last case hearing heard at?', with: 'Aldershot')
```

Comment:
Changing the expectation like this changes the entire test. The test says 'can select a value from the autocomplete', but what this test is describing is _not_ selecting a value from the autocomplete, but instead entering a custom value. I think this spec should now be 2 specs, one showing that if you enter a custom value, by default it remains a custom value. The other spec should show how you actually use autocomplete to get from entering 'Aldershot' to clicking on 'Aldershot Magistrates Court' to the value being recorded being "Aldershot Magistrates Court".